editoast/src/views/path/pathfinding.rs

```diff
@@ -278,15 +287,22 @@ pub async fn pathfinding_from_train_batch(
     train_schedules: &[TrainSchedule],
     rolling_stocks: &HashMap<String, RollingStockModel>,
 ) -> Result<Vec<PathfindingResult>> {
-    let mut results = vec![PathfindingResult::NotEnoughPathItems; train_schedules.len()];
+    let mut results = vec![
+        PathfindingResult::Failed(PathfindingFailed::InputError(
+            PathfindingInputError::NotEnoughPathItems
+        ));
+        train_schedules.len()
+    ];
```

Comment:
I believe keeping a `Vec<PathfindingInputError>` and performing the conversion into a `PathfindingResult` just before returning would be easier to read. Wdyt?

Reply:
As discussed, it doesn't bring much and adds a new conversion step. Resolving.